chore/copyright_update.go

```diff
@@ -0,0 +1,95 @@
+package chore
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/rs/zerolog/log"
+
+	"github.com/coreruleset/crs-toolchain/context"
+	"github.com/coreruleset/crs-toolchain/regex"
+)
+
+var logger = log.With().Str("component", "copyright-update").Logger()
+
+// CopyrightUpdate updates the copyright portion on the rules files to the provided year and version.
+func CopyrightUpdate(ctxt *context.Context, version string, year string) {
+	err := filepath.WalkDir(ctxt.RootDir(), func(path string, d fs.DirEntry, err error) error {
+		if err != nil {
+			// abort
+			return err
+		}
+		if d.IsDir() {
+			// continue
+			return nil
+		}
+		if strings.HasSuffix(d.Name(), ".conf") || strings.HasSuffix(d.Name(), ".example") {
+			if err := processFile(path, version, year); err != nil {
+				// abort
+				return err
+			}
+		}
+		// continue
+		return nil
+	})
+	if err != nil {
+		logger.Fatal().Err(err).Msg("failed to update copyright")
+	}
+}
+
+func processFile(filePath string, version string, year string) error {
+	logger.Info().Msgf("Processing %s", filePath)
+
+	contents, err := os.ReadFile(filePath)
+	if err != nil {
+		return err
+	}
+
+	output, err := updateRules(version, year, contents)
+	if err != nil {
+		return err
+	}
+
+	err = os.WriteFile(filePath, output, fs.ModePerm)
+	if err != nil {
```

Comment:
No need for the nil check here.

```suggestion
	return err
```

Reply:
Done.